src/index.html

```diff
@@ -22,6 +22,323 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!-- HEADER -->
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+        />
+      </a>
+
+      <nav class="navigation">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#apple"
+              class="is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#samsung"
+              class="nav_link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#smartphones"
+              class="nav_link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#laptops"
+              class="nav_link"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#gadgets"
+              class="nav_link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#tablets"
+              class="nav_link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="#photo"
```

Comment:
Ensure that the ID `photo` exists in the HTML or adjust the link to point to the correct section or page.